CONTRIBUTING.md

```diff
@@ -226,6 +226,36 @@ e.g. run tests against v 1.0
 
 If the "--api-version" argument is not specified, all runs act against default api version for each tested command.
 
+To run specific test:
+
+`pytest azext_iot/tests/central/test_iot_central_int.py::TestIotCentral::test_central_query_methods_run`
+
+Integration test environment variables needed:
+
+```
+env =
+    AZURE_TEST_RUN_LIVE=True
+    azext_iot_testrg=
+    azext_iot_teststorageaccount=
+    azext_iot_teststoragecontainer=
+    azext_iot_central_app_id=
+    azext_iot_central_scope_id=
+    azext_iot_central_token=
+    azext_iot_central_primarykey=
+    azext_iot_central_storage_cstring=
+    azext_iot_central_storage_container=
+```
+
+Not all of them are required. `azext_iot_teststoragecontainer` is the same as `azext_iot_central_storage_container`.
+
+If you have the error message when you start running integration tests:
+
+```
+ERROR    cli.azure.cli.core.azclierror:azlogging.py:212 (ResourceGroupNotFound) Resource group 'xxxxx' could not be found.
+```
+
+You need to set up your azure CLI subscription using `az account set --subscription xxxxxx` so CLI could find the resource group successfully.
```

Comment:
this is more of common knowledge - as in the tests will run with your current credentials (and account - including sub). You do not need to have this in the contributing since local hand testing (as in writing out the commands) wont work till you are in the right sub.

Reply:
Okay. I'll remove this.